ios/PrivacyInfo.xcprivacy

```diff
@@ -29,5 +29,7 @@
 			</array>
 		</dict>
 	</array>
+	<key>NSPrivacyCollectedDataTypes</key>
+    <array/>
```

Comment:
Not sure why Github is formatting it like this, but was added via Xcode and looks fine in VS Code so should be right.